platforms/js/build_js.py

```diff
@@ -150,6 +150,11 @@ def get_cmake_cmd(self):
         else:
             cmd.append("-DWITH_PTHREADS_PF=OFF")
 
+        if self.options.simd:
+            cmd.append("-DCV_ENABLE_INTRINSICS=ON")
+        else:
+            cmd.append("-DCV_ENABLE_INTRINSICS=OFF")
```

Comment:
We may also need to pass the compile flag `-msimd128` to compiler.

Reply:
yes, the base impl doesn't have any simd code, so I missed it. good catch. will add.